python/katana/native_interfacing/wrappers.py

```diff
@@ -86,7 +87,7 @@ def typeof(val, c):
 
         return Type
 
-    def register_method(self, func_name, typ, cython_func_name=None, addr=None, dtype_arguments=None):
+    def register_method(self, func_name, typ, cython_func_name=None, addr=None, dtype_arguments=None, data=None):
```

Comment:
who would call this function?

Reply:
This is called from register_method in numba_support.py and that is called from the C++ code.